pkg/cmd/ssh_aws.go

```diff
@@ -242,7 +246,7 @@ func (a *AwsInstanceAttribute) sshPortCheck() {
 		cmd := exec.Command("bash", "-c", ncCmd)
 		output, _ := cmd.CombinedOutput()
 		fmt.Println("=>", string(output))
-		if strings.Contains(string(output), "succeeded") {
+		if strings.Contains(string(output), "open") {
```

Comment:
instead of `sshPortCheck()`, can't we reuse the newly introduced [function]?(https://github.com/gardener/gardenctl/pull/445/files#diff-8a970d51dea43e926c0b913c26887464b362c5f723fb1d740a5a07d428434c07R355) 
```go
func CheckIPPortReachable(ip string, port string) error 
```

Reply:
thanks @petersutter , i add retry logic in `CheckIPPortReachable` function and use it here

Reply:
that's what I was asking in the other PR https://github.com/gardener/gardenctl/pull/445#discussion_r525999141
> and this works without having to dial/try multiple times within a certain amount of time?

Reply:
yes, previously i though it was working as retry but later i found it's not, so i added the logic